mobile/test/common/integration/test_server.cc

```diff
@@ -203,8 +203,14 @@ void TestServer::start(TestServerType type, int port) {
   }
   }
 
-  upstream_ = std::make_unique<AutonomousUpstream>(std::move(factory), port_, version_,
-                                                   upstream_config_, true);
+  if (version_ == Network::Address::IpVersion::v4) {
+    auto address = Network::Utility::parseInternetAddressNoThrow("127.0.0.3", port_);
```

Comment:
with the additional tests this was flaking 5% of the time with 127.0.0.1.  Passes cleanly with 127.0.0.3

Reply:
Do you know why that de-flakes the test?

Reply:
address in use, so someone else on that forge machine snagging it.  keep in mind for these tests we have to pick a port to listen on for the second server, rather than listen(0)
if it continues flaking, I'll do (a bunch of other refactors) to allow TCP to listen first as that's the one which is more likely to be on a time wait list